doc/examples/register_binary_fuzzy.py

```diff
@@ -0,0 +1,166 @@
+"""
+=========================================================
+Diffeomorphic Registration with binary and fuzzy images
+=========================================================
+This example for registering binary and fuzzy images together.
+This may be seen as aligning sensed (fuzzy) image to the
+and template (binary) image.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from skimage import draw, filters
+from dipy.align.imwarp import SymmetricDiffeomorphicRegistration
+from dipy.align.metrics import SSDMetric
+from dipy.viz import regtools
+
+"""
+Let's generate sample template image as combination of three ellipses.
+The fuzzy (sensed) is a smooth version of the reference image.
```

Comment:
=> "We will generate the fuzzy (sensed) version of the image by smoothing the reference image"